oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/SearchAppResultOptions.jsx

```diff
@@ -15,10 +14,8 @@ export const SearchAppResultOptions = ({ sortOptions, layoutOptions }) => {
   return (
     <Grid>
       <Grid.Row verticalAlign="middle">
-        <Grid.Column floated="left" textAlign="left" width={5}>
+        <Grid.Column textAlign="right" floated="right" width={13}>
```

Comment:
![Screenshot 2023-10-25 at 10 43 08](https://github.com/oarepo/oarepo-ui/assets/646743/28eebc2c-c11c-4802-b348-95316e703f02)

please increase padding between elements here, or make total results look like regular text (not label)

Reply:
@mirekys thanks pushed